src/NovaAdapterBase.sol

```diff
@@ -8,26 +8,26 @@ import {SafeTransferLib} from "@solmate/utils/SafeTransferLib.sol";
 abstract contract NovaAdapterBase is INovaAdapterBase {
     using SafeTransferLib for ERC20;
 
-    address immutable sDAI;
+    address immutable savings;
     address immutable asset;
```

Comment:
let's replace `asset` with `stable` so that we use just two variable names: `stable` and `savings`

Reply:
we also have `uint256 assets` for the amount to deposit, should I rename that too (to `amountInStable` for example) ?